varia/header.php

```diff
@@ -42,63 +30,6 @@
 <div id="page" class="site">
 	<a class="skip-link screen-reader-text" href="#content"><?php _e( 'Skip to content', 'varia' ); ?></a>
 
-	<?php if ( class_exists( 'A8C\FSE\WP_Template' ) ) : // If the FSE plugin is active, use the Header template for content. ?>
-
-		<header id="masthead" class="fse-template-part fse-header entry-content">
-			<?php
-				$template = new A8C\FSE\WP_Template();
-				$template->output_template_content( A8C\FSE\WP_Template::HEADER );
-			?>
-		</header>
-
-	<?php else : // Otherwise we'll fallback to the default Varia header below. ?>
-		<header id="masthead" class="<?php echo $header_classes; ?>" role="banner">
-			<?php get_template_part( 'template-parts/header/site', 'branding' ); ?>
-
-			<?php if ( $has_primary_nav && $has_primary_nav_items ) : ?>
-				<nav id="site-navigation" class="main-navigation" aria-label="<?php esc_attr_e( 'Main Navigation', 'varia' ); ?>">
-
-					<input type="checkbox" role="button" aria-haspopup="true" id="toggle" class="hide-visually">
-					<label for="toggle" id="toggle-menu" class="button">
-						<?php _e( 'Menu', 'varia' ); ?>
-						<span class="dropdown-icon open">+</span>
-						<span class="dropdown-icon close">&times;</span>
-						<span class="hide-visually expanded-text"><?php _e( 'expanded', 'varia' ); ?></span>
-						<span class="hide-visually collapsed-text"><?php _e( 'collapsed', 'varia' ); ?></span>
-					</label>
-
-					<?php
-					$main_nav_args = array(
-						'theme_location' => 'menu-1',
-						'menu_class'     => 'main-menu',
-						'items_wrap'     => '<ul id="%1$s" class="%2$s" aria-label="submenu">%3$s</ul>',
-					);
-					if ( get_theme_mod( 'enable_side_menu' ) === 1 ) {
-						$main_nav_args['container_class'] = 'main-menu-container';
-					}
-					wp_nav_menu( $main_nav_args );
-					?>
-				</nav><!-- #site-navigation -->
-			<?php endif; ?>
-
-			<?php if ( has_nav_menu( 'social' ) ) : ?>
-				<nav class="social-navigation" role="navigation" aria-label="<?php esc_attr_e( 'Social Links Menu', 'varia' ); ?>">
-					<?php
-					wp_nav_menu(
-						array(
-							'theme_location' => 'social',
-							'menu_class'     => 'social-links-menu',
-							'link_before'    => '<span class="screen-reader-text">',
-							'link_after'     => '</span>' . varia_get_icon_svg( 'link' ),
-							'depth'          => 1,
-						)
-					);
-					?>
-				</nav><!-- .social-navigation -->
-			<?php endif; ?>
-
-		</header><!-- #masthead -->
-
-	<?php endif; ?>
+	<?php get_template_part( 'template-parts/header/header', 'content' ); ?>
```

Comment:
Can we move this code here:

<?php 
if ( ( true === get_theme_mod( 'hide_site_header', false ) && is_front_page() && is_page() ) ) : return; // Return if this is the homepage and the hide-header setting is active. ?>

Then we don't need to repeat it in every template part.

Reply:
yes, I believe for the header we will be able to do that, I will go ahead with that, thx